Framework/Crystal/src/OptimizeCrystalPlacement.cpp

```diff
@@ -420,33 +421,12 @@ void OptimizeCrystalPlacement::exec() {
   setProperty("FitInfoTable", RRes);
 
   //----------- update instrument -------------------------
-
-  IPeak &peak = Peaks->getPeak(0);
-  boost::shared_ptr<const Instrument> OldInstrument = peak.getInstrument();
-  boost::shared_ptr<const ParameterMap> pmap_old =
-      OldInstrument->getParameterMap();
-  auto pmap_new = boost::make_shared<ParameterMap>();
-
-  PeakHKLErrors::cLone(pmap_new, OldInstrument, pmap_old);
-
-  double L0 = peak.getL1();
   V3D newSampPos(Results["SampleXOffset"], Results["SampleYOffset"],
                  Results["SampleZOffset"]);
 
-  boost::shared_ptr<const Instrument> Inst = OldInstrument;
-
-  if (OldInstrument->isParametrized())
-    Inst = OldInstrument->baseInstrument();
-
-  auto NewInstrument = boost::make_shared<Geometry::Instrument>(Inst, pmap_new);
-
-  SCDCalibratePanels::FixUpSourceParameterMap(NewInstrument, L0, newSampPos,
-                                              pmap_old);
-
-  for (int i = 0; i < OutPeaks->getNumberPeaks(); i++)
-    OutPeaks->getPeak(i).setInstrument(NewInstrument);
```

Comment:
It is not clear to me why are copy of `OldInstrument` was being made. Did the `PeakHKLErrors::cLone` call drop some parameters?

Do *all* peaks have the right instrument set without a call like this?